### DISCUSSION TITLE
Accessing raw element for a node

### DISCUSSION THREAD

**Discussion by dkulchenko** [formkit/formkit#761] [Ideas]
Is there a way to access the raw DOM element for a node? 

FormKit takes over the `ref=""` behavior from Vue which I would normally use to do so, and it doesn't look like there's any functions or fields on the FormKit `node` type that I could use to access the raw DOM element. Any ideas?

**Comment by fenilli** [comment]
There aren't functions to get the dom elements, you can use native JS for this using `getElementById` and adding an ID to the element you want to have the reference.
